common/src/main/java/io/homeassistant/companion/android/common/sensors/SensorReceiverBase.kt

```diff
@@ -102,6 +103,11 @@ abstract class SensorReceiverBase : BroadcastReceiver() {
             }
         }
 
+        if (intent.action == ACTION_STOP_BEACON_SCANNING) {
+            BluetoothSensorManager.enableDisableBeaconMonitor(context, false)
+            return
```

Comment:
If the stop action is used, should the sensor be allowed to update instead of returning before updates are sent?

Reply:
We actually update sensors inside the above method so this was added to prevent 2 runs

Reply:
Looked at the wrong function, ignore the previous comment 😅